docs/override_policy.md

```diff
@@ -0,0 +1,120 @@
+# Override Policy
+
+The [OverridePolicy][1] and [ClusterOverridePolicy][2] are used to declare override rules for resources when 
+they are propagating to different clusters.
+
+## Difference between OverridePolicy and ClusterOverridePolicy
+TBD
+## Resource Selector
+TBD
+## Target Cluster
+TBD
+## Overriders
+
+Karmada offers various alternatives to declare the override rules:
+- `ImageOverrider`: dedicated to override images for workloads.
+- `CommandOverrider`: dedicated to override commands for workloads.
+- `ArgsOverrider`: dedicated to override args for workloads.
+- `PlaintextOverrider`: a general-purpose tool to override any kind of resources.
+
+### ImageOverrider
+TBD
+### CommandOverrider
+The `CommandOverrider` is a refined tool to override commands(e.g.`/spec/template/spec/containers/0/command`) 
+for workloads, such as `Deployment`.
+
+The allowed operations are as follows:
+- `add`: appends one or more flags to the command list.
+- `remove`: removes one or more flags from the command list.
+
+#### Examples
+Suppose we create a deployment named `myapp`.
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: myapp
+  ...
+spec:
+  template:
+    spec:
+      containers:
+        - image: myapp
+          name: myapp
+          command:
+            - ./myapp
+            - --parameter1=foo
+            - --parameter2=bar
+```
+
+**Example 1: Add flags when workloads are propagating to specific clusters.**
+```yaml
+apiVersion: policy.karmada.io/v1alpha1
+kind: OverridePolicy
+metadata:
+  name: example
+spec:
+  ...
+  overriders:
+    commandOverrider:
+      - containerName: alpine
```

Comment:
Maybe the `containerName` should be `myapp` not `alpine`

Reply:
Done